src/manifest.lisp

```diff
@@ -49,6 +62,19 @@ does not exist."
            (return-from manifest-get-by-name matches)))))
 
 
+(defun manifest-update ()
+  "Upgrade the Qi manifest by checking out the latest version from git."
+  (cond ((probe-file (fad:merge-pathnames-as-file +manifest-directory+ "manifest.lisp"))
+         (format t "~%---> Upgrading Qi manifest~%")
+         (run-git-command "fetch origin" +manifest-directory+)
+         (run-git-command "rebase origin/master" +manifest-directory+))
```

Comment:
Actually, this will cause problems if you have uncommitted changes in the manifest.  Alternatively we could

- just `git pull`
- `git stash`, then rebase, then `git pop`
- `git reset --hard origin/master`, blowing up local modifications